fdbserver/worker.actor.cpp

```diff
@@ -1178,6 +1178,7 @@ UpdateWorkerHealthRequest doPeerHealthCheck(const WorkerInterface& interf,
 		double lastLoggedTime = peer->lastLoggedTime <= 0.0 ? peer->lastConnectTime : peer->lastLoggedTime;
 
 		TraceEvent(SevDebug, "PeerHealthMonitor")
+		    .suppressFor(5.0)
```

Comment:
FYI for other readers: other TraceEvent change was changing SevDebug -> SevVerbose for sim testing. We don't want that here because in sim testing, SevVerbose is not printed, but we still want PeerHealthMonitor events.